app/portal/chapters/[id]/page.js

```diff
@@ -121,13 +123,19 @@ const ChapterDetailPage = () => {
   const handleOpenCourseLink = () => setOpenCourseLink(true)
   const handleCloseCourseLink = () => setOpenCourseLink(false)
   // Selected Chapter Data
-  const { chapters, course_permissions } = useData()
+  const { chapters, course_permissions, setChapters } = useData()
   const [selectedChapter, setSelectedChapter] = useState(null)
   const [selectedCourses, setSelectedCourses] = useState(null)
+  // Edit Chapter Data
+  const [editChapterData, setEditChapterData] = useState(null)
   // Member NEW modal
   const [openMemberNew, setOpenMemberNew] = useState(false)
   const handleOpenMemberNew = () => setOpenMemberNew(true)
   const handleCloseMemberNew = () => setOpenMemberNew(false)
+  // Axios
+  const axiosInstance = AxiosWithAuth()
+  // Router
+  const router = useRouter()
```

Comment:
@paulstgermain with the introduction of `window.location` is `router` being used at all? Line 184 uses it, but is also commented out.